src/controller/article.py

```
@@ -54,3 +58,259 @@ async def create_article_ctrl(
+            raise HTTPException(
+                403, detail="You are not allowed to write to this board"
+            )
+
+        article.title = body.title
+        article.board_id = body.board_id
+        article.updated_at = datetime.now(timezone.utc)
+        try:
+            self.session.commit()
+        except IntegrityError:
+            self.session.rollback()
+            raise HTTPException(status_code=409, detail="unique field already exists")
```

Ruff (0.14.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)